README.md

```diff
@@ -44,8 +44,10 @@
 ## Quickstart
 
 ```shell
-# with npx
+# with npx and the github repo
 npx react-native init MyApp --template=https://github.com/echobind/react-native-template
```

Comment:
Should we pitch the GH option as the "unreleased" latest and greatest? I think its best to:
- Have users install using `npx react-native init MyApp --template=@echobind/react-native-template`
- If that fails (due to error like you mentioned): use `npx react-native init --ignore-existing` (this should only be for folks with existing RN installs and should get less common over time
- If you want to test out the latest and greatest: GH

Reply:
Great idea! Let me update that now